### PR TITLE
6 assessment put route

### PR DESCRIPTION
**Pull request by paul-renwick**
Passing tests, functioning in postman.

### CODE REVIEW

File: tests/server/routes/auth.test.js [hunk 1]
```diff
@@ -19,4 +19,4 @@ test.skip('POST /api/v1/register adds a new user', () => {
       expect(res.body.message).toBe('Authentication successful.')
       expect(res.body.token).not.toBeNull()
     })
-})
+})
```
Comment (emilyparkes) [inline tests/server/routes/auth.test.js:22]: remove changes to this file please.

File: tests/server/routes/submitAssessmentRoute.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const request = require('supertest')
```
Comment (emilyparkes) [inline tests/server/routes/submitAssessmentRoute.test.js]: Please change the name of this file to match the name of the file you are testing.
e.g tests/server/routes/assessments.test.js

File: tests/server/db/submitAssessmentDB.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const env = require('../../test-environment')
```
Comment (emilyparkes) [inline tests/server/db/submitAssessmentDB.test.js]: Please also keep the naming of the file the same as you would find it in server/db

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: tests/server/routes/submitAssessmentRoute.test.js (hunk 2 above)
Comment (paul-renwick) [reply tests/server/routes/submitAssessmentRoute.test.js]: Done

File: tests/server/db/submitAssessmentDB.test.js (hunk 3 above)
Comment (paul-renwick) [reply tests/server/db/submitAssessmentDB.test.js]: Done

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**